quickfixj-core/src/main/java/quickfix/mina/AbstractIoHandler.java

```diff
@@ -42,10 +46,17 @@
     protected final Logger log = LoggerFactory.getLogger(getClass());
     private final NetworkingOptions networkingOptions;
     private final EventHandlingStrategy eventHandlingStrategy;
+    private final SessionSettings sessionSettings;
+    private boolean logMessageWhenSessionNotFound;
 
-    public AbstractIoHandler(NetworkingOptions options, EventHandlingStrategy eventHandlingStrategy) {
+    public AbstractIoHandler(SessionSettings settings, NetworkingOptions options, EventHandlingStrategy eventHandlingStrategy) throws FieldConvertError, ConfigError {
+        sessionSettings = settings;
         networkingOptions = options;
         this.eventHandlingStrategy = eventHandlingStrategy;
+        logMessageWhenSessionNotFound = true;
+        if (sessionSettings.isSetting(Session.SETTING_LOG_MESSAGE_WHEN_SESSION_NOT_FOUND)) {
+            logMessageWhenSessionNotFound = sessionSettings.getBool(Session.SETTING_LOG_MESSAGE_WHEN_SESSION_NOT_FOUND);
```

Comment:
Sorry, one more thing: please catch and ignore `ConfigError` and `FieldConvertError` here and hence fall back to the default `logMessageWhenSessionNotFound = true` in that case. That will also make the extension of the `throws` clause (that you needed to do in various classes) unnecessary.

Reply:
Sorry, @chrjohn, will change this, too - as requested.

Reply:
No problem, thanks. :+1:

Reply:
@chrjohn this is done now, sorry for the delay